code/game/objects/items/tajara.dm

```diff
@@ -135,6 +135,7 @@
 	icon_state = "adhomai_clock"
 	item_state = "adhomai_clock"
 	contained_sprite = TRUE
+	var/static/months = list("Menshe-aysaif","Sil'nryy-aysaif","Menshe-rhazzimy","Sil'nryy-rhazzimy")
```

Comment:
This is a tiny nitpick, but I _think_ we usually style stuff with spaces after commas?

```suggestion
	var/static/months = list("Menshe-aysaif", "Sil'nryy-aysaif", "Menshe-rhazzimy", "Sil'nryy-rhazzimy")
```